src/spec/page-objects/settings-page.js

```diff
@@ -57,5 +58,13 @@ class SettingsPage {
       return val[0].click();
     });
   }
+  getDisabledDomain(){
+    return this._domainsDisabled.getText();
+  }
+  clickDisabledDomain(){
+    return this._domainsDisabled.then(function(val){

```

Comment:
Right now i'm mocking only 1 disable button, should we consider to test when having more than 1?